thank-you-2017/heart.js

```diff
@@ -0,0 +1,17 @@
+$('.sponsorsBox').magnificPopup({
```

Comment:
it would be great if you could use the existing structure we have for libraries, css and javascript. I know it's a lot of cleaning up work, but I think it would really be best to have everything in the respective folders (`assets/javascripts`, `assets/stylesheets`) and to move the images as well to the `img` folder. Could you do that?
Ideally also sticking to the convention of using scss :)